components/retro-go/rg_system.c

```diff
@@ -381,7 +389,7 @@ rg_app_t *rg_system_init(int sampleRate, const rg_handlers_t *handlers, const rg
     atexit(&exit_handler);
     rtc_time_init();
 
-    xTaskCreate(&system_monitor_task, "sysmon", 2560, NULL, RG_TASK_PRIORITY, NULL);
```

Comment:
This should be undone, I think you didn't pull changes I've made before pushing.